### ISSUE TITLE
The clr-validate-icon does not have a textual equivalent

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3891]
URL:
https://clarity.design/documentation/input

WCAG 2.1 Guideline: 
4.1.2 Name, Role, Value 
(Level A)

Description:
The clr-validate-icon does not have a textual equivalent. This icon is not decorative, it is used as a non-color based indication of an error state. 
 
According to the 
Error state section of the Inputs documentation page, the clr-validate-icon appears when the .clr-error class is added to the .clr-control-container element. When the clr-validate-icon appears it does not include a textual equivalent as seen below.

<clr-icon class="clr-validate-icon" shape="exclamation-circle" role="none">

Recommendation:
Since the Clarity consumer does not control the attributes of the clr-validate-icon, Clarity will have to provide the textual equivalent with aria-label as seen below.

<clr-icon aria-label='error' class="clr-validate-icon" shape="exclamation-circle" role="none">

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
